frontend-project/src/typings.d.ts

```diff
@@ -28,9 +28,11 @@ interface Window {
   ) => void;
   reloadAuthorized: () => void;
 }
-
 declare let ga: Function;
 
+declare let build_date: date;
```

Comment:
To chyba powinni być constem

Reply:
przeszło mi to przez myśl, następnie z jakiegoś powodu projekt mi się posypał i ta myśl się gdzieś zapodziała.
poprawione :)

Reply:
Robił to bodajże @ad-m, ja nie mam nic przeciwko zmianie na UPPER_CASE, wydaje mi się, że zadziała to pozytywnie na czytelność